modules/osd-create-cluster-ccs.adoc

```diff
@@ -151,7 +151,7 @@ endif::osd-on-aws[]
 ifdef::osd-on-gcp[]
 encryption keys through the Google Cloud Key Management Service.
 endif::osd-on-gcp[]
-The key is used for encryption of persistent volumes in your cluster.
```

Comment:
Are the control plane data volumes encrypted with the KMS keys in this scenario? If so, you could add the following line in place of this. By removing this line and not adding another description, it might not be clear to the reader what the key is used for in this.

Also, line 147 states ".. Optional: Select *Encrypt persistent volumes with customer keys* if you want to provide your own...". Should that be removed or described differently also to highlight the data volumes only?

Reply:
Updated, thank you!